frontend/src/pages/org/crawls-list.ts

```diff
@@ -272,84 +272,85 @@ export class CrawlsList extends LiteElement {
-                >
-              `
-            )}
-          </sl-select>
-        </div>
-
-        <div class="col-span-12 md:col-span-3 flex items-center">
-          <div class="whitespace-nowrap text-neutral-500 mr-2">
-            ${msg("Sort by:")}
+          <div class="grow flex items-center">
+            <div class="whitespace-nowrap text-neutral-500 mr-2 ml-2">
+              ${msg("Sort by:")}
+            </div>
+            <div class="grow flex">
+            <sl-select
+              class="flex-1 md:w-[9.2rem]"
+              placement="bottom-end"
+              distance="4"
+              size="small"
+              pill
+              value=${this.orderBy.field}
+              @sl-select=${(e: any) => {
+                const field = e.detail.item.value as SortField;
+                this.orderBy = {
+                  field: field,
+                  direction:
+                    sortableFields[field].defaultDirection ||
+                    this.orderBy.direction,
+                };
+              }}
+            >
+              ${Object.entries(sortableFields).map(
+                ([value, { label }]) => html`
+                  <sl-menu-item value=${value}>${label}</sl-menu-item>
+                `
+              )}
+            </sl-select>
+            <sl-icon-button
+              name="arrow-down-up"
+              label=${msg("Reverse sort")}
+              @click=${() => {
+                this.orderBy = {
+                  ...this.orderBy,
+                  direction: this.orderBy.direction === "asc" ? "desc" : "asc",
+                };
+              }}
+            ></sl-icon-button>
+            </div>
           </div>
-          <sl-select
-            class="flex-1"
-            placement="bottom-end"
-            distance="4"
-            size="small"
-            pill
-            value=${this.orderBy.field}
-            @sl-select=${(e: any) => {
-              const field = e.detail.item.value as SortField;
-              this.orderBy = {
-                field: field,
-                direction:
-                  sortableFields[field].defaultDirection ||
-                  this.orderBy.direction,
-              };
-            }}
-          >
-            ${Object.entries(sortableFields).map(
-              ([value, { label }]) => html`
-                <sl-menu-item value=${value}>${label}</sl-menu-item>
-              `
-            )}
-          </sl-select>
-          <sl-icon-button
-            name="arrow-down-up"
-            label=${msg("Reverse sort")}
-            @click=${() => {
-              this.orderBy = {
-                ...this.orderBy,
-                direction: this.orderBy.direction === "asc" ? "desc" : "asc",
-              };
-            }}
-          ></sl-icon-button>
-        </div>
       </div>
 
       ${this.userId
-        ? html` <div class="my-1 text-right">
+        ? html` <div class="h-6 mt-2 flex flex-row-reverse">
```

Comment:
You can achieve the alignment with [`justify-content: end`](https://tailwindcss.com/docs/justify-content) instead of row-reverse. flex-row-reverse will reverse the order of all flex child items.
```suggestion
        ? html` <div class="h-6 mt-2 flex justify-end">
```